packages/api/src/webhooks/cannon.ts

```diff
@@ -626,6 +623,48 @@ export default class WebhookCannon {
   }
 }
 
+export async function resendWebhook(
```

Comment:
Hmmmm, so the resending is synchronous? I thought we would just send an event over to RabbitMQ and let the cannon do a regular delivery, with retries etc. 

If it is synchronous (and blocking), shouldn't the API return a 200 with the new `WebhookResponse` in the payload instead?

Reply:
As discussed on a call let's keep it blocking but I'll return the response directly 👍